DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>I suggest adding a sentence or two here indicating the accuracy of the different methods. E.g. <code>temp_yflux_adv_int_z</code> is an online diagnostic and thus is accurate (except of course that it only captures the advective components, not the parameterized components). <code>net_sfc_heating</code>  is an approximate calculation that only applies if the ocean is in steady state. Likewise, using <code>ty_trans</code>,<code>temp</code> is approximate as it neglects contributions from time varying correlations between transport and temperature at frequencies faster than the output frequency of the diagnostics themselves.</p><p> </p><p>You shouldn't need <code>dxt</code> and <code>dzt</code> for these calculations, since you're using the transport <code>ty_trans</code> not velocity.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
For the `temp_yflux_adv` MHT, when you say that it does not measure the parameterised components, it can also be understood that it doesn't capture the MHT due to bolus transport, which I think is also advective. So, isn't this statement better: `it only captures the advective components, not the parameterised diffusive components`?

Reply:
<p>Yes, hence you want to say "only captures resolved advective components, not the parameterised diffusive or advective components.".</p>

Reply:
Okay, I understand now, thanks!